brand/templates/admin/brand/commentary/change_form.html

```diff
@@ -0,0 +1,34 @@
+{% extends "admin/change_form.html" %}
+{% load i18n admin_urls %}
+
+{% block object-tools-items %}
+{{ block.super }}
+<li>
+    <a href="{% url 'admin:refresh_harvest_data' original.pk %}" class="refreshlink">
+        {% trans "Refresh Harvest Data" %}
```

Comment:
I see the data displaying but don't see the refresh button.

Reply:
It is in each detail view as a button. And it is an admin actino in the commentary overview, to let people select all or a selection of brands which should refresh.

I assume this is ok and let it this way.